README.md

```diff
@@ -101,20 +101,20 @@ To get a local copy up and running follow these steps.
 1. Connect to an instance of Dewy
     ```typescript
     import { Dewy } from 'dewy_ts';
-    const dewy = new Dewy()
+    const dewy_kb = new Dewy().kb
```

Comment:
Only mention `.kb` once? Not sure if we want this, since it makes it less useful if we ever have another tag. Could change back to `dewy.kb.addDocument` (same number of characters).

Reply:
Yeah this seems like a good approach

Reply:
Which one? `dewy_kb.addDocument` or `dewy.kb.addDocument`?

Reply:
I may be leaning back towards `dewy.kb.addDocument` just because it avoids an indirection (`dewy` is the client) and it's the same length, so it's not really saving anything.